servers/server_wrap_mt_common.h

```diff
@@ -40,6 +40,12 @@
 #define MAIN_THREAD_SYNC_CHECK
 #endif
 
+// Macros for wrapping multithreaded server calls.
+// - FUNC{n} is a straightforward call w/no return type that takes n arguments (implicitly C given that n is 0).
```

Comment:
I don't really understand what "implicitly C given that n is 0" means?
```suggestion
// - FUNC{n} is a straightforward call with no return type that takes n arguments (implicitly C given that n is 0).
```